src/components/Clusters/views/ClusterOverview/ClusterDetails.js

```diff
@@ -1,12 +1,18 @@
+import { useMemo } from 'react';
 import { useTranslation } from 'react-i18next';
-import { ClusterStorageType } from '../ClusterStorageType';
+import { useRecoilValue } from 'recoil';
+
 import { useGetGardenerProvider } from './useGetGardenerProvider';
 import { useGetVersions } from './useGetVersions';
 import { useFeature } from 'hooks/useFeature';
+import { kymaResourcesAtom } from '../../../../state/kymaResourcesAtom';
```

Comment:
Please check if this import works, as far as I remember it should
```suggestion
import { kymaResourcesAtom } from 'state/kymaResourcesAtom';
```